spynl/tests/test_exception.py

```diff
@@ -35,6 +40,46 @@ def make_response(self):
 
     return app
 
+class ToBeMapped(Exception):
+    """
+    dummy exception that stands in for an exception from spynl.models
+    """
+    def __init__(self, extra):
+        self.extra = extra
+
+    def __str__(self):
+        return 'A message from spynl.models'
+
+
+@SpynlException.this_exception_maps(ToBeMapped)
+class Mapped(SpynlException):
+    """
+    corresponding SpynlException exception to ToBeMapped
+    """
+    http_excalate_as = HTTPConflict
+
+    def __init__(self, raised_exception):
+        message = 'This is a Spynl message'
```

Comment:
I suggest we put a uuid – e.g. '6a7e46a3-2879-4ad5-88ae-169dbd349831'. "This is a Spynl message" is rather generic.